src/util.py

```diff
@@ -0,0 +1,185 @@
+"""Service classes for OS interaction and multithreading."""

```

Comment:
@bhavikapanara most of the code in this util.py file is not required for this repo. Please clean up unused code.

Reply:
done